DfE.FindInformationAcademiesTrusts/Services/Export/ExportService.cs

```diff
@@ -67,73 +89,245 @@ private static byte[] GenerateSpreadsheet(
         using var workbook = new XLWorkbook();
         var worksheet = workbook.Worksheets.Add("Academies");
 
-        // Adding Trust name and type 
-        worksheet.Cell(1, 1).Value = trustSummary?.Name ?? string.Empty;
-        worksheet.Row(1).Style.Font.Bold = true;
-        worksheet.Cell(2, 1).Value = trustSummary?.Type ?? string.Empty;
+        WriteTrustInformation(worksheet, trustSummary);
+        WriteHeaders(worksheet, headers, 3);
```

Comment:
3 is the same as the default value so is not needed
```suggestion
        WriteHeaders(worksheet, headers);
```

Same on line 217